src/Link.tsx

```diff
@@ -35,11 +35,11 @@ const hoverColor = system({
 })
 
 const Link = styled.a<StyledLinkProps>`
-  color: ${props => (props.muted ? get('colors.gray.6')(props) : get('colors.blue.5')(props))};
+  color: ${props => (props.muted ? get('colors.text.secondary')(props) : get('colors.text.link')(props))};
```

Comment:
Oops. I already updated Link in https://github.com/primer/components/pull/1089

Reply:
Ah, my fault! My eyes lined up with the wrong row on the spread sheet! I'll scrub this commit out and force push.